### DISCUSSION TITLE
Startle trial rejection

### DISCUSSION THREAD

**Discussion by laurenpattyn** [bachlab/PsPM#412] [Q&A] [answered]
Dear,

I did a fear conditioning experiment with an electrical CS and US. The US follows the CS after 6 seconds in the experimental group and 15 seconds in the control group. A startle probe is presented either 3-4 seconds or 13-14 seconds after the CS. To asses fear learning, I will use the trial by trial amplitude estimates obtained by the GLM for SCR and startle response in PSPM.

For the startle response analysis, I was wondering if any manual trial rejection before applying the GLM is recommended? For manual peak scoring, each trial that has noise in the baseline or a spontaneous blink during the startle sound presentation is rejected and this trial would be set as a missing value. Is this also done/needed with model-based approaches?

Thank you!
Lauren

**Comment by dominikbach** [comment, marked as answer]
Dear Lauren

model-based analysis distinguishes blink artefacts from true starte responses to some extent by their different onset and shape. It might still improve signal quality to do manual artefact rejection; this has not been systematically tested so far.

Dominik
